index.html

```diff
@@ -1458,11 +1462,26 @@ <h2 id="OfflineAudioContext">
                   start rendering <code>length</code> sample-frames of audio
                   into <var>buffer</var>.
                   </li>
-                  <li>For every render quantum, check and suspend the rendering
-                  if necessary.
+                  <li>At the beginning of every render quantum, check scheduled
+                  suspensions and suspend the rendering if necessary. When the
+                  rendering is suspended,
+                    <ol>
+                      <li>Resolve the promise associated with the suspension.
+                      </li>
+                      <li>Queue a task to fire an event named
+                      <code>statechange</code> at this instance.
+                      </li>
+                    </ol>
                   </li>
                   <li>If a suspended context is resumed, continue to render the
-                  buffer.
+                  buffer. When the context is resumed,
+                    <ol>
+                      <li>Resolve the promise associated with the resume.
+                      </li>
+                      <li>Queue a task to fire an event named
+                      <code>statechange</code> at this instance.
+                      </li>
+                    </ol>
```

Comment:
This is already documented in `BaseAudioContext` specs for suspension/resumption (I believe that did not exist at the time this PR was created).